contracts/GameJamManager.sol

```diff
@@ -0,0 +1,79 @@
+pragma solidity ^0.5.11;
+
+import "openzeppelin-solidity/contracts/access/Roles.sol";  //Import Roles to implement custom Role Based Access Control
+
+
+import "./GameJam.sol";
+import "./interfaces/IRegistry.sol";
+
+
+contract GameJamManager {
+    using Roles for Roles.Role;
+
+    // Define a ContractRegistry to be setup in the Constructor
+    IRegistry registry;
+    // Define a mapping to track GameJams using their name with their address
+    mapping (bytes32 => address) public gameJamList;
+
+    // Define RBAC List, Modifiers and Checks
+    Roles.Role private _gameJamManager;
```

Comment:
Should we be looking to extract the common Role logic that is in each contract or is it fine this way ?

Reply:
It's fine this way, I think we're using the abstraction that `openzeppelin-solidity` built